conformance/lang/expressions/unary-expression/unary-minus-expression.balt

```diff
@@ -0,0 +1,893 @@
+Test-Case: output
+Description: Test unary negative expression for positive integers, no overflow scenarios.
+Label: DecimalNumber, int, unary-minus
+
+public function main() {
+    int a = 1;
+    int b = 101;
+    int c = 0;
+    int d = 9223372036854775807;
+    io:println(-a); // @output -1
+    io:println(-b); // @output -101
+    io:println(-c); // @output 0
+    io:println(-d); // @output -9223372036854775807
+}
+
+Test-Case: output
+Description: Test unary negative expression for negative integers, no overflow scenarios.
+Label: DecimalNumber, int, unary-minus
+
+public function main() {
+    int a = -1;
+    int b = -101;
+    int c = 0;
+    int d = -9223372036854775807;
+    io:println(-a); // @output 1
+    io:println(-b); // @output 101
+    io:println(-c); // @output 0
+    io:println(-d); // @output 9223372036854775807
+}
+
+Test-Case: panic
+Description: Test unary negative overflow for int.
+Labels: DecimalNumber, int, unary-minus
+
+public function main() {
+    int a = -9223372036854775808;
```

Comment:
Shall we use `int:MIN_VALUE` instead of `-9223372036854775808`? 
```ballerina
    int a = int:MIN_VALUE;
    int b = -a;
```
This also doesn't panic ATM.

Reply:
Yeah, have created an issue. Will add it

Reply:
updated in 6862c07